Sources/SwiftGodotMacroLibrary/MacroGodot.swift

```diff
@@ -477,3 +490,42 @@ struct godotMacrosPlugin: CompilerPlugin {
         SignalMacro.self
     ]
 }
+
+private extension DeclGroupSyntax {
+    var exportCategories: [MacroExpansionDeclSyntax] {
+        memberBlock.members.compactMap { $0.decl.as(MacroExpansionDeclSyntax.self) }
+    }
+}
+
+private extension MacroExpansionDeclSyntax {
+    private var isExportCategory: Bool {
+        macroName.text == "exportCategory"
+    }
+    
+    var exportCategoryPrefix: String? {
+        guard isExportCategory, let exportCategoryName else { return nil }
+        return exportCategoryName.camelCaseToSnakeCase().replacingOccurrences(of: " ", with: "_") + ":"
```

Comment:
I see no reason why the prefix couldn't just be the category name, this might be over complicating it, altho I like how it reads.